lldb/test/API/lang/swift/async/taskgroups/TestSwiftTaskGroupSynthetic.py

```diff
@@ -0,0 +1,26 @@
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+
+
+class TestCase(TestBase):
+
+    @swiftTest
+    def test(self):
+        """Print a TaskGroup and verify its children."""
+        self.build()
+        lldbutil.run_to_source_breakpoint(
+            self, "break here", lldb.SBFileSpec("main.swift")
+        )
+        self.expect(
+            "v group",
+            substrs=[
```

Comment:
While this probably covers it, It would be slightly better to test GetChildWithName/GetChildAtIndex/GetNumChildren separately, since they are implemented separately, too.

Reply:
will do

Reply:
Added a new test in https://github.com/swiftlang/llvm-project/pull/10143/commits/31845cf2fa735a791ba02bb602a80f7be1ae462b

Reply:
Good call, I had a StringRef bug in `GetIndexOfChildWithName`